docs/language/crypto.md

```diff
@@ -19,6 +19,10 @@ pub enum HashAlgorithm: UInt8 {
 
     /// SHA3_384 is Secure Hashing Algorithm 3 (SHA-3) with a 384-bit digest.
     pub case SHA3_384 = 4
+
+    /// KMAC_128_BLS_BLS12381 is a variant of KMAC_128 hashing algorithm, that
```

Comment:
I have suggested `KMAC_128_BLS_BLS12381` but I'm re considering this choice and I think `KMAC128_BLS_BLS12381` is more concise and slightly easier to read. 

The original algo name is KMAC128 (no space). I initially added the underscore to mirror `SHAx_256`. The underscore there might be necessary because `SHA2256` or `SHA3384` is hard to read, but we don't have this problem with `KMAC128`. 

I should be doing the same with the crypto library . Sorry for the late change.

Reply:
Yeah, that makes sense. Changed the `KMAC_128_*` to `KMAC128_*` in fffaebb.